tests/phpunit/tests/image/editorImagick.php

```diff
@@ -935,4 +935,67 @@ public static function data_alpha_transparency_is_preserved_after_resize() {
 			),
 		);
 	}
+
+	/**
+	 * Test AVIF quality filters.
+	 *
+	 * @ticket 61614
+	 *
+	 * Temporarily disabled until we can figure out why it fails on the Trixie based PHP container.
+	 * See https://core.trac.wordpress.org/ticket/63932.
+	 * @requires PHP < 8.3
+	 */
+	public function test_quality_with_avif_conversion_file_sizes() {
+		$temp_dir = get_temp_dir();
+		$file     = $temp_dir . '/33772.jpg';
+		copy( DIR_TESTDATA . '/images/33772.jpg', $file );
+
+		$editor = wp_get_image_editor( $file );
+		// Only continue if the server supports AVIF.
+		if ( ! $editor->supports_mime_type( 'image/avif' ) ) {
```

Comment:
this is probably wrong. The editor chosen for the jpeg is not the same as the editor chosen to output the avif.

We added code (possibly after this test was added) that chooses the editor to use based on the _output format_ since the `image_editor_output_format` filter is used to output avifs for this test, the actual editor used will be the one that supports AVIF. That said, if the filter is applied so only one engine is available it will be fine.